syft/frameworks/torch/pointers/pointer_tensor.py

```diff
@@ -165,6 +165,9 @@ def attr(self, attr_name):
         self.__setattr__(attr_name, attr_ptr)
         return attr_ptr
 
+    def dim(self) -> int:
```

Comment:
I just had to fix this for multipointer tensor lol

Reply:
I saw it and fixed here as well, thanks btw! :1st_place_medal: